server/project-service/README.md

```diff
@@ -331,13 +598,20 @@ system, including creating, updating, and deleting projects, as well as managing
   POST /project/api/projects/66c60ba518da486b1e9c08d5/workflows
   [
    {
-            "id": "efc14b0f-1848-4dc7-ad0d-69ef6d7f0d9c",
-            "type": "workflow"
+        "id": "efc14b0f-1848-4dc7-ad0d-69ef6d7f0d9c",
```

Comment:
Without specifying the "workflow" or "execution", the type field is null.

Reply:
Addressed